web/src/urls.ts

```diff
@@ -15,6 +15,8 @@ export const DEPARTMENT_URL = `${config.apiBaseUrl}/api/department`;
 export const ROLE_URL = `${config.apiBaseUrl}/api/role`;
 export const LOCATION_URL = `${config.apiBaseUrl}/api/location`;
 export const INSPECTION_LOCATION_URL = `${config.apiBaseUrl}/api/inspection-location`;
+export const DATA_INGESTION_URL = `${config.apiBaseUrl}/api/data-ingestion`;
+export const DATA_INGESTION_SOURCE_URL = `${config.apiBaseUrl}/api/data-ingestion-source`;
```

Comment:
Prefer the api pattern used in this project https://github.com/icefoganalytics/wrap/blob/9da5e31931351ce911d441556caa388d4dc2ac35/web/src/api/delegations-api.ts as it makes tracing the code much easier. It also creates a healthy level of isolation, and indirection, between your API, and your use of that API, throughout the codebase.